docs/symfony/get-it-started.md

```diff
@@ -71,65 +73,78 @@ class PaymentToken extends Token
 
 ```php
 <?php
-namespace Acme\PaymentBundle\Entity;
+// src/EntityPayment.php
+
+namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 use Payum\Core\Model\Payment as BasePayment;
 
 /**
- * @ORM\Table
  * @ORM\Entity
+ * @ORM\Table
  */
 class Payment extends BasePayment
 {
     /**
-     * @ORM\Column(name="id", type="integer")
      * @ORM\Id
-     * @ORM\GeneratedValue(strategy="IDENTITY")
-     *
-     * @var integer $id
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
      */
     protected $id;
 }
 ```
 
-next, you have to add mapping information, and configure payum's storages:
+Next, you have to add mapping information, and configure payum's storages:
 
 ```yml
-#app/config/config.yml
+# config/packages/payum.yaml (Smyonfy >=4) or app/config/config.yml (Symfony <=3)
 
 payum:
     security:
         token_storage:
-            Acme\PaymentBundle\Entity\PaymentToken: { doctrine: orm }
+            App\Entity\PaymentToken: { doctrine: orm }
 
     storages:
-        Acme\PaymentBundle\Entity\Payment: { doctrine: orm }
+        App\Entity\Payment: { doctrine: orm }
             
     gateways:
         offline:
             factory: offline
 ```
 
-_**Note**: You can add other gateways to the gateways section too._
+For Symfony >=4, you need to add this to your `config/services.yaml`:
+```yml
+services:
+    Payum\Core\Payum:
```

Comment:
this could be added to the PayumBundle.

Reply:
Cool, could you do that?

Reply:
Done :-) See https://github.com/Payum/PayumBundle/pull/482